test/test_imports.py

```diff
@@ -1,8 +1,27 @@
-# Import ElectricPy modules just to make sure they load correctly
+# test all imports in Gemotry folder
```

Comment:
If we're still testing importing `electricpy` (which we should) this comment should more broadly express that we're testing that all modules import without error.